### DISCUSSION TITLE
Aruco Motion Capture

### DISCUSSION THREAD

**Discussion by marios-stam** [USC-ACTLab/crazyswarm#536] [Q&A]
Hello, I have currently developed a "motion capture" system based on Aruco Markers placed on top of the Crazyflies. I was wondering if there is a way to integrate it as seperate motion capture system .

**Comment by jpreiss** [comment]
You could contribute the code to https://github.com/IMRCLab/libmotioncapture. This is the motion capture abstraction layer used by Crazyswarm.
